__tests__/tag.test.js

```diff
@@ -111,18 +112,28 @@ describe('Tag', () => {
     expect(tag.getDecoratedComponentInstance().state.isDragging).to.be.true;
     const el = TestUtils.findRenderedDOMComponentWithTag(root, 'span');
     expect(el.className).contains('opacity-none');
+    expect(el.className).contains('cursor-move');
     backend.simulateEndDrag();
     expect(el.className).not.contains('opacity-none');
     expect(tag.getDecoratedComponentInstance().state.isDragging).to.be.false;
   });
 
-  test('should not be draggable if readOnly is true', () => {
-    const root = TestUtils.renderIntoDocument(mockItem({ readOnly: true }));
-    const backend = root.getManager().getBackend();
-    const tag = TestUtils.findRenderedComponentWithType(root, Tag);
-    backend.simulateBeginDrag([
-      tag.getDecoratedComponentInstance().getHandlerId(),
-    ]);
-    expect(tag.getDecoratedComponentInstance().state.isDragging).to.be.false;
+  [
+    { overrideProps: { readOnly: true }, title: 'readOnly is true' },
+    { overrideProps: { allowDragDrop: false }, title: 'allowDragDrop is false' },
+  ].forEach((data) => {
+    const { title, overrideProps } = data;
+    test(`should not be draggable when ${title}`, () => {
+      const root = TestUtils.renderIntoDocument(mockItem({...overrideProps}));
```

Comment:
we have started migrating our test to enzyme instead of test utils, can you rewrite this test using `enzyme` ?
here is a reference spec you can check https://github.com/prakhar1989/react-tags/blob/fe0e0d55447876122e34a18ff4943eccfe4dd49b/__tests__/reactTags.test.js#L280